File: Lancement d'un programme exec/exo1.py
import sys

def main():
    if len(sys.argv) != 3:
        print(f"Usage: {sys.argv[0]} <entier> <entier>")
        sys.exit(1)

    num1 = int(sys.argv[1])
    num2 = int(sys.argv[2])
    somme = num1 + num2
    
    print(f"La somme de {num1} et {num2} est : {somme}")

if __name__ == "__main__":
    main()

File: Lancement d'un programme exec/exo1.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) 
{
    if(argc != 3)
    {
       printf("Usage: %s <entier> <entier>\n", argv[0]);
       return 1;
    }
    int num1 = atoi(argv[1]);
    int num2 = atoi(argv[2]);
    int sum = num1 + num2;
    printf("La somme de %d et %d est : %d\n", num1, num2, sum);
    return 0;
}